bears/vcs/git/GitMergeInspectBear.py

```diff
@@ -0,0 +1,38 @@
+from bears.vcs.VCSCommitBear import COMMIT_TYPE
+from bears.vcs.git.GitCommitBear import GitCommitBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class GitMergeInspectBear(GlobalBear):
```

Comment:
It isnt useful to have a bear which only has a default disabled option -- by default it does nothing.

Lets call this `GitLinearCommitBear` and remove the `allow_git_merge_commit` option.

Add a parameter for the help message URL, default to
http://www.bitsnbites.eu/a-tidy-linear-git-history/

coala will use this bear.
coala will change the url to http://api.coala.io/en/latest/Developers/Git_Basics.html#rebasing